cmd/remove.go

```diff
@@ -32,7 +32,9 @@ func NewRemoveCommand() *cmdr.Command {
 }
 
 func remove(cmd *cobra.Command, args []string) error {
-
+	if cmd.Flag("nix").Changed {
+		return removePackage(cmd, args)
```

Comment:
I think these Nix functions should be moved to their respective commands to remove all `cmd/nix_[cmd].go` files.